tv_apps/app-deck/build/build.js

```diff
@@ -4,10 +4,25 @@
 
 var utils = require('utils');
 
+var AppDeckBuilder = function() {};
+
+// XXX: we should extract the logic of copying default-cards.json
+// to one common place. See https://bugzil.la/1123585
+AppDeckBuilder.prototype.execute = function(options) {
+  var stageDir = utils.getFile(options.STAGE_APP_DIR);
+  var configFile = utils.getFile(stageDir.path, 'js', 'init.json');
+  var defaultConfigFile = utils.getFile(options.GAIA_DIR, 'tv_apps',
+                                'tv_shared', 'resources', 'default-cards.json');
+  var defaultConfig = utils.getJSON(defaultConfigFile);
```

Comment:
Do or may we support customization?
An example can be found at other apps, like:
```
  var distDir = options.GAIA_DISTRIBUTION_DIR;
  var customize = JSON.parse(utils.getDistributionFileContent('gallery',
                this.DEFAULT_VALUE, distDir));
```